examples/reference/widgets/Tabulator.ipynb

```diff
@@ -0,0 +1,724 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Frozen rows\n",
+    "\n",
+    "Another common scenario is when you have certain rows with special meaning, e.g. aggregates which summarize the information in the rest of the table. In this case you may want to freeze those rows so they do not scroll out of view. You can achieve this by setting a list of `frozen_rows` by integer index (which can be positive or negative):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "date_df = pd.util.testing.makeTimeDataFrame().iloc[:10]\n",
+    "agg_df = pd.concat([date_df, date_df.median().to_frame('Median').T, date_df.mean().to_frame('Mean').T])\n",
+    "agg_df.index= agg_df.index.map(str)\n",
+    "\n",
+    "pn.widgets.tables.Tabulator(agg_df, frozen_rows=[-2, -1], width=400)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Grouping\n",
+    "\n",
+    "Another useful option is the ability to group specific rows together, which can be achieved using `groups` parameter. The `groups` parameter should be composed of a dictionary mapping from the group titles to the column names:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pn.widgets.tables.Tabulator(date_df, width=400, groups={'Group 1': ['A', 'B'], 'Group 2': ['C', 'D']})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pagination\n",
+    "\n",
+    "When working with large tables we sometimes can't send all the data to the browser at once. In these scenarios we can enable pagination, which will fetch only the currently viewed data from the server backend. This may be enabled by setting `pagination='remote'` and the size of each page can be set using the `page_size` option:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "large_df = pd.util.testing.makeCustomDataframe(100000, 5) \n",
+    "\n",
+    "paginated_table = pn.widgets.tables.Tabulator(large_df, pagination='remote', page_size=10)\n",
+    "paginated_table"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering\n",
+    "\n",
+    "A very common scenario is that you want to attach a number of filters to a table in order to view just a subset of the data. You can achieve this through callbacks or other reactive approaches but the  `.add_filter` method makes it much easier.\n",
+    "\n",
+    "#### Widget filters\n",
+    "\n",
+    "The simplest approach to filtering is simply to declare one or more widgets and then using the `.add_filter` to add them as filters on specific columns. The filtering will depend on the type of the widget or parameters value:\n",
```

Comment:
```suggestion
    "Perhaps the simplest approach to filtering is simply to declare one or more widgets and then using the `.add_filter` to add them as filters on specific columns. The filtering will depend on the type of the widget or parameter value:\n",
```

(Personally, I'd think the _simplest_ approach to filtering is to supply a static value, e.g. `make='Ford'`; that seems simpler than a widget!)

Reply:
New text:

> The simplest approach to filtering is to select along a column with a constant or dynamic value. The `.add_filter` method allows passing in constant values, widgets and parameters. If a widget or parameter is provided the table will watch the object for changes in the value and update the data in response.

Reply:
Ok